expo-maps/src/Map.tsx

```diff
@@ -120,6 +142,51 @@ export class ExpoMap extends React.Component<ExpoMapViewProps> {
               type: 'kml',
               filePath: filePath.localUri,
             } as KMLObject;
+          } else if (Utils.isGeoJson(child)) {
+            if (child.props.defaultStyle?.marker?.color != undefined) {
+              if (typeof child.props.defaultStyle?.marker.color !== 'number') {
+                child.props.defaultStyle.marker.color =
+                  Utils.mapColorToNativeMarkerColor(
+                    child.props.defaultStyle.marker.color
+                  );
+              }
+            }
+
+            if (
+              child.props.defaultStyle?.polygon?.fillColor != undefined &&
+              !Utils.isHexColor(child.props.defaultStyle.polygon.fillColor)
+            ) {
+              child.props.defaultStyle.polygon.fillColor =
+                Utils.mapColorToHexColor(
+                  child.props.defaultStyle.polygon.fillColor as Color
+                );
+            }
+
+            if (
+              child.props.defaultStyle?.polygon?.strokeColor != undefined &&
+              !Utils.isHexColor(child.props.defaultStyle.polygon.strokeColor)
+            ) {
+              child.props.defaultStyle.polygon.strokeColor =
+                Utils.mapColorToHexColor(
+                  child.props.defaultStyle.polygon.strokeColor as Color
+                );
+            }
+
+            if (
+              child.props.defaultStyle?.polyline?.color != undefined &&
+              !Utils.isHexColor(child.props.defaultStyle.polyline.color)
+            ) {
+              child.props.defaultStyle.polyline.color =
+                Utils.mapColorToHexColor(
+                  child.props.defaultStyle.polyline.color as Color
+                );
+            }
```

Comment:
I think it could be extracted to a private method so it would be more readable, but it's just styling so I leave it up to you

Reply:
Sounds like good idea 👍